Source/engine.cpp

```diff
@@ -291,31 +270,10 @@ void CelBlitLightSafe(BYTE *pDecodeTo, BYTE *pRLEBytes, int nDataSize, int nWidt
 			if (!(width & 0x80)) {
 				i -= width;
 				if (dst < gpBufEnd && dst > gpBufStart) {
-					if (width & 1) {
-						dst[0] = tbl[src[0]];

```

Comment:
This broke object rendering, tbl[] is the lighting table so everything was fully lit.